.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,139 @@
+name: Maintenance
+on:
+    workflow_dispatch:
+        inputs:
+            signing-key-id:
+                required: true
+                type: string
+                default: C8B77BCBE16CD2B94B43F9C8757397B82D4ED7B0
+
+jobs:
+    create-branch-prefix:
+        uses: ./.github/workflows/create_branch_prefix.yml
+        with:
+            prefix: 'maintenance'
+            version: $(jq -r '.version' package.json)
+            signing-key-id: ${{ inputs.signing-key-id }}
+        secrets:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }}
+
+    prepare-maintenance:
+        name: Prepare Maintenance
+        needs:
+            - create-branch-prefix
+        runs-on: ubuntu-latest
+        outputs:
+            test-infos: ${{ steps.get-test-infos.outputs.test-infos }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ needs.create-branch-prefix.outputs.base-branch }}
+                  token: ${{ secrets.LASTMJS_GITHUB_TOKEN }}
+
+            - uses: ./.github/actions/setup_node
+
+            - run: npm install
+
+            # Run global prettier/lint fixes
```

Comment:
Use our new fancy scripts to do the pretty and lint stuff

Reply:
I changed my mind, these are great and maybe even preferable